airflow/providers/airbyte/operators/model.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any, Dict, List
+
+from openlineage.client.facet import ColumnLineageDatasetFacet
+from openlineage.client.generated.schema_dataset import SchemaDatasetFacet
+from openlineage.client.run import Dataset
+
+from airflow.providers.openlineage.extractors import OperatorLineage
```

Comment:
In general, we should not import OpenLineage in top level imports as described [here](https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/guides/developer.html#how-to-properly-implement-openlineage-methods). It's sometimes being done for the whole module, but then this module itself is locally imported within get_openlineage_facets_ method so it's not breaking anything. We need to make sure that if somebody has no openlineage installed the code will still work.

Reply:
oh now it makes sense why in some places its imported in the scope of the function, thanks for the explanation !